app/views/approach_stages/show.html.erb

```diff
@@ -20,7 +22,7 @@
 
         <% unless @featured_approaches.count.zero? %>
           <div class="featured-contributors">
-            <h6 class='zero-margin-bottom'>Featured Contributors</h6>
+            <h6 class='zero-margin-bottom'><%= t '.contributors' %></h6>
```

Comment:
Very clever usage of the ["lazy" lookup](http://guides.rubyonrails.org/i18n.html#lazy-lookup) style here. Is there a reason that we couldn't use this in other places?